CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
+## 3.57.2
```

Comment:
Let's leave this as unreleased for now, since it's not out yet (and a new version probably won't go out until after the holidays)

```suggestion
## unreleased
```

Just an FYI though, when this does go out, we'd bump the minor version because it's a new feature, not just a bug fix. So the version will likely be `3.58.0`